src/ui_parts/settings_menu.gd

```diff
@@ -286,6 +286,13 @@ func setup_content() -> void:
 			add_dropdown(Translator.translate("UI scale"), dropdown_values, dropdown_map)
 			add_advice(Translator.translate("Changes the scale factor for the interface."))
 			
+			current_setup_setting = "ui_max_fps"
+			add_number_dropdown(Translator.translate("Max FPS"),
+					[0, 30, 60, 120, 240], true, false,
+					SaveData.UI_MAX_FPS_MIN, SaveData.UI_MAX_FPS_MAX)
+			add_advice(Translator.translate(
+					"Limits the framerate of the user interface. If set to 0, the framerate will be unlimited"))
```

Comment:
Needs punctuation at the end.

I would prefer if this is documented on the dropdown itself (e.g. the 0 option being "Unlimited" instead), but this is fine for now.